docs/operations/images.md

```diff
@@ -1,8 +1,8 @@
 # Images
 
-As of Kubernetes 1.27 the default images used by kOps are the **[official Ubuntu 22.04](#ubuntu-2204-jammy)** images.
+As of Kubernetes 1.27 the default images used by kOps are the x86_64 (amd64) **[official Ubuntu 22.04](#ubuntu-2204-jammy)** images. If you want to pull the default kOps image, but for an arm64 processor architecture, you can specify an arm64-based instance name in the --control-plane-size and --node-size parameters of the kops create cluster command.
```

Comment:
The topic here is how to find the right images and I don't think this change adds more clarity.

Reply:
I agree, especially in light of the edits you made to commands, adding a wildcard in the proc-architecture position of the search string. 

My thought process in adding this note was to try to help folks bring up their ARM64 instances without editing config (which is super helpful when you are just getting started and learning). I am OK with it being nixed, nonetheless. With the edit to the search strings, folks should be able to read between the lines on how to instantiate ARM images.